src/components/character-count/template.njk

```diff
@@ -0,0 +1,36 @@
+{% from "../error-message/macro.njk" import govukErrorMessage -%}
+{% from "../hint/macro.njk" import govukHint %}
+{% from "../label/macro.njk" import govukLabel %}
+{% from "../textarea/macro.njk" import govukTextarea %}
+
+{#- a record of other elements that we need to associate with the input using
+   aria-describedby – for example hints or error messages -#}
+{% set describedBy = "" %}
+
+<div class="govuk-character-count" data-module="character-count"
+{% if params.maxlength %} data-maxlength="{{ params.maxlength }}"{% endif %}
```

Comment:
Non-blocking, if we used the whitespace stripping syntax these would end up on the same line when the markup is rendered.

```html
<div class="govuk-character-count" data-module="character-count"
data-maxlength="blah"
></div>
```

vs


```html
<div class="govuk-character-count" data-module="character-count" data-maxlength="blah"></div>
```